transforms/em-to-ed/util/add-import-specifier.js

```diff
@@ -0,0 +1,39 @@
+const FORMATTING = require('./formatting');
+
+function addImportSpecifier(j, code, importSpecifier, source) {
```

Comment:
Helper to add an import specifier. For example, take the following scenarios where we want to add the import specifier of `alias` from `@ember/object/computed`.

```js
//BEFORE
import { or } from '@ember/object/computed';

//AFTER
import { or, alias } from '@ember/object/computed';

//BEFORE
// import from @ember/object/computed does not exist

//AFTER
import { alias } from '@ember/object/computed';
```